rpc/README.md

```diff
@@ -3,230 +3,31 @@
-    "method": "_compute_transaction_hash",
-    "params": [
-        {
-            "deps": [],
-            "inputs": [
-                {
-                    "args": [],
-                    "previous_output": {
-                        "block_hash": "0xca97a033aa9765914bc0b7887ff8723037614df11d06783555ff97e9a0e72b55",
-                        "cell": {
-                            "index": "0",
-                            "tx_hash": "0x602b713fac38c0224bc2e8ba0ec9c8363d7d1e508897b33c32f45d7554883c81"
-                        }
-                    },
-                    "since": "0"
-                }
-            ],
-            "outputs": [
-                {
-                    "capacity": "100000000000",
-                    "data": "0x",
-                    "lock": {
-                        "args": [],
-                        "code_hash": "0x0000000000000000000000000000000000000000000000000000000000000001"
-                    },
-                    "type": null
-                }
-            ],
-            "version": "0",
-            "witnesses": []
-        }
-    ]
-}' \
-| tr -d '\n' \
-| curl -H 'content-type:application/json' -d @- \
-http://localhost:8114
-```
-
-```json
-{
-    "id": 2,
-    "jsonrpc": "2.0",
-    "result": "0xbecff6ee8cf2a40bdf391a03df501791b7946cea9a8d83b3d328a1ea5e1bc000"
-}
-```
-
-### `dry_run_transaction`
-
-Dry run transaction and return the execution cycles. 
-
-This method will not check the transaction validaty, but only run the lock script 
-and type script and than return the execution cycles.
-
-
-#### Examples
-
-```bash
-echo '{
-    "id": 2,
-    "jsonrpc": "2.0",
-    "method": "dry_run_transaction",
-    "params": [
-        {
-            "deps": [],
-            "inputs": [
-                {
-                    "args": [],
-                    "previous_output": {
-                        "block_hash": "0xca97a033aa9765914bc0b7887ff8723037614df11d06783555ff97e9a0e72b55",
-                        "cell": {
-                            "index": "0",
-                            "tx_hash": "0x602b713fac38c0224bc2e8ba0ec9c8363d7d1e508897b33c32f45d7554883c81"
-                        }
-                    },
-                    "since": "0"
-                }
-            ],
-            "outputs": [
-                {
-                    "capacity": "100000000000",
-                    "data": "0x",
-                    "lock": {
-                        "args": [],
-                        "code_hash": "0x0000000000000000000000000000000000000000000000000000000000000001"
-                    },
-                    "type": null
-                }
-            ],
-            "version": "0",
-            "witnesses": []
-        }
-    ]
-}' \
-| tr -d '\n' \
-| curl -H 'content-type:application/json' -d @- \
-http://localhost:8114
-```
-
-```json
-{
-    "id": 2,
-    "jsonrpc": "2.0",
-    "result": {
-        "cycles": "0"
-    }
-}
-```
+* [`chain`](#chain)
```

Comment:
As commented in the original PR, the title link is broken, and the name should match the one in config file, a.k.a, capitalizing it.

Reply:
Oops. I didn't see any comments in https://github.com/nervosnetwork/ckb/pull/785

Updated rpc.py: capitalize module name, and link module to the first API inside it.